Game.cpp

```diff
@@ -57,7 +57,8 @@ Game::StartWatching(BMessenger *target)
 bool
 Game::StopWatching(BMessenger *target)
 {
-	return fTargets.erase(std::remove(fTargets.begin(), fTargets.end(), target));
+	fTargets.erase(std::remove(fTargets.begin(), fTargets.end(), target));
+	return true;
```

Comment:
Thanks for the feedback! :)

I'll change the function to returning `void` - I thought it was an API compatibility issue that made you need to use `bool`.

```c
fTargets.erase(std::remove(fTargets.begin(), fTargets.end(), target), fTargets.end());
```